lib/bundle/brewfile.rb

```diff
@@ -5,19 +5,28 @@ module Brewfile
     module_function
 
     def path(dash_writes_to_stdout: false)
-      if ARGV.include?("--global")
-        Pathname.new("#{ENV["HOME"]}/.Brewfile")
-      else
-        filename = ARGV.value("file")
-        if filename == "-"
-          filename = if dash_writes_to_stdout
-            "/dev/stdout"
-          else
-            "/dev/stdin"
+      bundle_file = "#{ENV["HOMEBREW_BUNDLE_FILE"]}"
```

Comment:
```suggestion
      bundle_file = ENV["HOMEBREW_BUNDLE_FILE"]
```

I think I'd prefer this to be named `env_bundle_file` or the like to make its origin clear since it is not mutated later.

Reply:
I like the rename to `env_bundle_file`.

I added the interpolation to prevent having to distinguish between `nil` vs `''` (empty string). That being said I didn't look into whether `ENV['FOO']` would return `nil` in cases where `FOO` was not set.

Reply:
I could implement this as `ENV.fetch("HOMEBREW_BUNDLE_FILE", "")` which might be more clear. It would also make it easier to mock, as well as test for `nil` vs `""` (empty string).

Reply:
> I added the interpolation to prevent having to distinguish between nil vs '' (empty string).

Reply:
> I added the interpolation to prevent having to distinguish between nil vs '' (empty string).

Can use `.blank?` instead.

> That being said I didn't look into whether `ENV['FOO']` would return `nil` in cases where `FOO` was not set.

It does.

Reply:
Sad days `undefined method `blank?' for "":String`. I'll try `.fetch()`

Reply:
See https://github.com/Homebrew/homebrew-bundle/pull/502#discussion_r293596295 regarding problems using `.blank?` or `.present?`. If these can be used I'm happy to change from `ENV.fetch()` to `ENV[]`.